scripts/generate-docs-types-versions.js

```diff
@@ -0,0 +1,72 @@
+const fs = require("fs");
```

Comment:
Do we need this file? Ideally we would only need to generate docs for the new current version in the future.